src/backend/base/langflow/schema/data.py

```
@@ -169,8 +169,11 @@
         files = self.data.get("files", [])
         if sender == MESSAGE_SENDER_USER:
             if files:
+                from langflow.schema.image import get_file_paths
+                
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)